railties/lib/rails/test_help.rb

```diff
@@ -31,6 +31,10 @@
   ActiveSupport.on_load(:action_dispatch_integration_test) do
     self.fixture_path = ActiveSupport::TestCase.fixture_path
   end
+else
+  ActiveSupport.on_load(:active_support_test_case) do
```

Comment:
What do you think of this as a solution? https://gist.github.com/ghiculescu/3e7a1f40cdae0617e7530027ce876f12

It removes the duplicated code, instead it just wraps the bits that interact with AR in if statements.

Reply:
`fixture_path` is defined in `ActiveRecord::TestFixtures`, so there is a need to add more conditions. Not sure which variant will become better.

Reply:
Ahh good point, this is probably better then.